files/default/upsert_outbound_proxy.groovy

```diff
@@ -0,0 +1,48 @@
+import groovy.json.JsonSlurper
+
+def params = new JsonSlurper().parseText(args)
+if (params.http["auth"]) {
```

Comment:
OPT: if http is not present the error risks to be unclear :D

Reply:
Updated the README, http is mandatory.